src/pages/clients.tsx

```diff
@@ -35,11 +40,71 @@ const ClientsPage: React.FC = () => {
             (major parts of) their web applications.
           </LargeText>
           <CaptionText>Not just a set of banners or a microsite.</CaptionText>
-          <CardGridContainer>TODO: CardGrid</CardGridContainer>
         </GridItem>
       </Grid>
+      <ImageCardGrid>
+        <GridItem xs={12} sm={4}>
+          <InternalLink to="/">
+            <ImageCard
+              alt="Lewandowski"
+              image={data.imagePlaceholder}
+              largeTitle="Client One"
+            />
+          </InternalLink>
+        </GridItem>
+        <GridItem xs={12} sm={4}>
+          <InternalLink to="/">
+            <ImageCard
+              alt="Lewandowski"
+              image={data.imagePlaceholder}
+              largeTitle="Client Two"
+            />
+          </InternalLink>
+        </GridItem>
+        <GridItem xs={12} sm={4}></GridItem>
```

Comment:
to make the item invisible it's more clear if you make it `xs=0` instead of `12`

Reply:
how do i then implement two consecutive empty grid items? when i change the code to  
![Screenshot 2020-06-25 at 10 58 39](https://user-images.githubusercontent.com/31735645/85689347-d7e94d80-b6d2-11ea-86db-c78646f16b64.png)
it breaks the layout, there is now only one empty grid item, not two
![Screenshot 2020-06-25 at 10 59 19](https://user-images.githubusercontent.com/31735645/85689507-fb13fd00-b6d2-11ea-9d48-6466e5aa275e.png)

Reply:
Like you did it already was correct 😉 what I meant is: the `xs=12` GridItem is empty. So it doesn't take any space. You can therefore also make it `xs=0`, which hides the whole item. The result will (should) be the same, for the code makes it more clear that this item is only there to give some spacing (`sm=4`) on larger devices.

Reply:
Aaah right, ok, i get it! Changed it to xs={0} s={4} 👍